### DISCUSSION TITLE
How do I add a custom paclage manager for fastfetch to list the ammount of packages?

### DISCUSSION THREAD

**Discussion by dnkmmr69420** [fastfetch-cli/fastfetch#820] [Q&A]
I want to have fastfetch list the ammount of packages of brew packages in a brew distrobox container using the command `cbrew`. Here is the [project](https://github.com/dnkmmr69420/brew-container) so you have an idea what I am talking about. cbrew uses the exact same syntax as brew but it runs it inside a distrobox container.

I tried [this](https://github.com/fastfetch-cli/fastfetch/issues/813#issuecomment-2081638138) and it didn't work at all

**Comment by CarterLi** [comment]
Use `--show-errors` to print errors

**Reply by dnkmmr69420** [reply]
got this
```
Feature Title: poll(&pollfd, 1, timeout) timeout
```

**Reply by CarterLi** [reply]
The error message means that child processing timed out. You need to increase the time limit with `--processing-timeout 10000`

See `fastfetch -h processing-timeout`

**Comment by dnkmmr69420** [comment]
there is no documentation for this as well

**Reply by CarterLi** [reply]
`fastfetch --list-modules | grep -i command` and `fastfetch -h | grep -i command`

**Comment by dnkmmr69420** [comment]
I got this for my output

```
deck@fedora:/var/home/deck$ fastfetch --list-modules | grep -i command
9)  Command       : Running custom shell scripts
deck@fedora:/var/home/deck$ fastfetch -h | grep -i command
  -h, --help <?command>: Show this message, or help for a specific command
      --gen-config <?path>: Generate a config file to specified path with options specified in the command line (if any)
      --command-shell <str>: Set the shell program to execute the command text
      --command-key <str>: Set the module key to display
      --command-text <str>: Set the command text to be executed
All options can be made permanent with command "fastfetch <options> --gen-config"
```

**Reply by mfgbhatti** [reply]
Hey,
I was looking to display my `systemd-analyze` output. I solved it after searching and few trials and lots of errors. I came to your question to find an answer to my problem. So, here is what it should be

```
        {
            "type": "command",
            "shell": "/usr/bin/bash", // Your shell path
            "text": "cbrew", // Command that will be executed in above shell 
            "key": "Packages", // Your text to display
            "format": "Installed packages {1}" // This will be the result of above command.
        }
````
